sqlalchemy_redshift/commands.py

```diff
@@ -425,9 +425,10 @@ def __init__(self, to, data_location, access_key_id=None,
                     '"ignore_header" parameter should be an integer'
                 )
 
-        if format not in self.formats:
+        if format is not None and str(format).upper() not in self.formats:
             raise ValueError('"format" parameter must be one of %s' %
                              self.formats)
+            format = format.upper()
 
         if compression is not None:
             if compression not in self.compression_types:
```

Comment:
is the plan to also change compression types?

Reply:
I can do that too, sure. Are they case-insensitive as well?

Reply:
no they're currently case sensitive. Remember this should simply be making a pythonic interface to the underlying query. It should not expose all the underlying implementation details of redshift

Reply:
Pythonic or not I'm not sure if it's a good idea to make our interface case-insensitive in places redshift is case-sensitive.